packages/loader/container-loader/src/container.ts

```diff
@@ -1018,9 +1018,10 @@ export class Container
 		);
 
 		const offlineLoadEnabled =
-			(this.isInteractiveClient &&
-				this.mc.config.getBoolean("Fluid.Container.enableOfflineLoad")) ??
-			options.enableOfflineLoad === true;
+			this.isInteractiveClient &&
+			((this.mc.config.getBoolean("Fluid.Container.enableOfflineLoad") ??
+				this.mc.config.getBoolean("Fluid.Container.enableOfflineFull")) === true ||
+				options.enableOfflineLoad === true);
```

Comment:
The config setting used to be able to trump `options.enableOfflineLoad` which is what I'm used to seeing - now you can't disable offline via config if the option says it's enabled.  Is that more desirable behavior for some reason?